.rubocop_todo.yml

```diff
@@ -1,93 +1,107 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2023-02-17 10:57:31 UTC using RuboCop version 1.12.1.
+# on 2024-08-19 09:25:53 UTC using RuboCop version 1.64.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 1
-# Cop supports --auto-correct.
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyleAlignWith.
+# SupportedStylesAlignWith: either, start_of_block, start_of_line
+Layout/BlockAlignment:
+  Exclude:
+    - 'Gemfile'
```

Comment:
This is what I highlighted so please regenerate after

Reply:
done